pom.xml

```diff
@@ -187,7 +187,7 @@
                 <configuration>
                     <cloneProjectsTo>${project.build.directory}/it</cloneProjectsTo>
                     <localRepositoryPath>${project.build.directory}/local-repo</localRepositoryPath>
-                    <settingsFile>src/it/settings.xml</settingsFile>
+                    <!-- settingsFile>src/it/settings.xml</settingsFile -->
                     <goals>
```

Comment:
Just to make the tests work in my environment. Quick hack for me, Do not merge this, sorry.